charts/stable/nextcloud/templates/_hpb.tpl

```diff
@@ -73,8 +73,22 @@ command:
 
     {{- if .Values.imaginary.enabled }}
     echo  "Imaginary High Performance Previews enabled, enabling it on Nextcloud..."
+    php /var/www/html/occ config:system:set enabledPreviewProviders 0 --value='OC\Preview\PNG'  # This might need to be an array like https://github.com/nextcloud/server/blob/master/config/config.sample.php#L1183
+    php /var/www/html/occ config:system:set enabledPreviewProviders 1 --value='OC\Preview\JPEG'
+    php /var/www/html/occ config:system:set enabledPreviewProviders 2 --value='OC\Preview\GIF'
+    php /var/www/html/occ config:system:set enabledPreviewProviders 3 --value='OC\Preview\BMP'
+    php /var/www/html/occ config:system:set enabledPreviewProviders 4 --value='OC\Preview\XBitmap'
+    php /var/www/html/occ config:system:set enabledPreviewProviders 5 --value='OC\Preview\MP3'
     php /var/www/html/occ config:system:set enabledPreviewProviders 6 --value='OC\Preview\Imaginary'
+    php /var/www/html/occ config:system:set enabledPreviewProviders 7 --value='OC\Preview\MarkDown'
+    php /var/www/html/occ config:system:set enabledPreviewProviders 8 --value='OC\Preview\OpenDocument'
+    php /var/www/html/occ config:system:set enabledPreviewProviders 9 --value='OC\Preview\TXT'
+    php /var/www/html/occ config:system:set enabledPreviewProviders 10 --value='OC\Preview\Krita'    
     php /var/www/html/occ config:system:set preview_imaginary_url --value='http://127.0.0.1:9090'
+    php /var/www/html/occ config:system:set preview_max_x --value='2048'
```

Comment:
Probably also make those user defined. As you will find ton of posts people trying to modify preview sizes etc.

Reply:
I just wanted it to be there first, but if we do add it to our config then the users can do as they please haha